playwright-tests/e2e/tests/comments.spec.ts

```diff
@@ -0,0 +1,106 @@
+//comments-spec.ts
+import { expect } from "@playwright/test";
+import { test } from "../fixtures/index";
+import { Users } from "../../constants";
+import { TaskPage } from "../pom/tasks";
+import { Auth } from "../pom/login";
+import { faker } from "@faker-js/faker";
+import LoginPage from "../pom/login";
+
+test.describe("Comment Page", () => {
+  let todoName: string;
+  let assigneeComment: string;
+  let assignerComment: string;
+
+  test.beforeEach(async ({ page }) => {
+    todoName = faker.word.verb(5);
+    assigneeComment = faker.lorem.sentence();
+    assignerComment = faker.lorem.sentence();
+  });
+
+  test("comment on task as creator and assigned user", async ({
+    page,
+    loginPage,
+    browser,
+    taskPage,
+  }) => {
+    page.goto("/");
+
+    await test.step("step 1 - create a new task by assigner", async () => {
+      await taskPage.createTaskAndVerify({
+        taskName: todoName,
+        userName: "Sam Smith",
+      });
+    });
+
+    await test.step("step 2 - go to that newely created task", async () => {
+      await page.getByTestId("tasks-pending-table").getByText(todoName).click();
+
+      const endPoint =
+        todoName.split(" ").length > 1
+          ? todoName.split(" ").join("-")
+          : todoName;
+
+      await page.waitForResponse(response => response.url().includes(endPoint));
+      expect(await page.locator("h1").innerText()).toBe(todoName);
+    });
+
+    await test.step("step 3 - create comment as assigner in the todo and verify", async () => {
+      await taskPage.createCommentAndVerify({
+        comment: assignerComment,
+        taskName: todoName,
+      });
+    });
```

Comment:
I did try to do that but it is showing some errors  because of not awaiting,

Reply:
![image](https://github.com/sreehari2003/granite-playwright/assets/66205494/18b76ffe-33f2-44ad-9dc7-e33a2aeb999e)

Reply:
its working fine now , the error was I was not directly returning the function


instead of this

```ts
 await test.step("step 1 - create a new task by assigner", () =>
      taskPage.createTaskAndVerify({
        taskName: todoName,
        userName: "Sam Smith",
      }));

```
I was doing this
```js
  await test.step("step 1 - create a new task by assigner", () => {
      taskPage.createTaskAndVerify({
        taskName: todoName,
        userName: "Sam Smith",
      });
    });

```